maps/away/ships/tcfl_patrol/tcfl_peacekeeper_ship.dm

```diff
@@ -2,7 +2,7 @@
 	name = "TCFL Corvette"
 	description = "Serving as the very foundation of the SCC's (And more specifically, NanoTrasen's) fleet of asset protection vessels, the Cetus-class is versatile and durable, but also clumsy and somewhat underpowered in regards to its engine and propulsion. It features small weapon hardpoints in its thruster arms, and a massive hangar host to the design's interdiction counterpart - the Hydrus-class shuttle. This one appears to be a Decanus-class, the Tau Ceti Foreign Legion variation of the design."
 	suffixes = list("ships/tcfl_patrol/tcfl_peacekeeper_ship.dmm")
-	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_BADLANDS)
+	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_BADLANDS, SECTOR_WEEPING_STARS, SECTOR_BURZSIA)
```

Comment:
I really don't see why the TCFL would be active in Coalition space.

Reply:
Specifically, the Weeping Stars region is said to be patrolled by both the TCFL and Coalition Rangers to protect trade routes passing through the area.